services/wallet/router/pathprocessor/processor_swap_paraswap.go

```diff
@@ -52,6 +52,8 @@ func getPartnerAddressAndFeePcnt(chainID uint64) (common.Address, float64) {
 		return common.HexToAddress("0x9a8278e856C0B191B9daa2d7DD1f7B28268E4DA2"), partnerFeePcnt
 	case walletCommon.BaseMainnet:
 		return common.Address{}, 0
```

Comment:
wtf why don't we have the address set for Base
this should be set to `0x107E3208A27e2A56D420fE6f8c5B88c821052f89` as per https://www.notion.so/Paraswap-0f4cd0acbc0e4a3f99bd21a4409c158b?pvs=4#17c8f96fb65c8028b4baf67198d26983